profiles/common/modules/custom/nexteuropa_token/modules/nexteuropa_token_ckeditor/nexteuropa_token_ckeditor.module

```diff
@@ -196,3 +196,15 @@ function nexteuropa_token_ckeditor_token_filter_filter_tokens($text, $filter, $f
 
   return _token_filter_filter_tokens($text, $filter, $format, $langcode, $cache, $cache_id);
 }
+
+/**
+ * Implements hook_custom_theme().
+ */
+function nexteuropa_token_ckeditor_custom_theme() {
+  if (strpos(current_path(), 'nexteuropa/token-ckeditor') === 0) {
+    $theme = variable_get('node_admin_theme', '0') ?
+      variable_get('admin_theme_default', 'seven') :
+      variable_get('theme_default', 'ec_resp');
```

Comment:
Before the PR 570, the code here was 
`if (strpos(current_path(), 'nexteuropa/token-ckeditor') === 0) {
    return 'seven';
  }`

The use of this value `variable_get('admin_theme_default', 'seven')` does not bother me so much because it is commonly used in all sites but `variable_get('theme_default', 'ec_resp')` bothers me because we cannot be sure that the default theme is compatible with the feature or Am I wrong?